src/Data/Vec/Bounded/Base.agda

```diff
@@ -80,3 +126,31 @@ zip = zipWith _,_
 
 align : ∀ {n} → Vec≤ A n → Vec≤ B n → Vec≤ (These A B) n
 align = alignWith id
+
+-- take and drop
+
+take : ∀ {m} n → Vec≤ A m → Vec≤ A (n ⊓ m)
+take             zero    _                = []
+take             (suc n) (Vec.[] , p)     = []
```

Comment:
Do we need to qualify the `[]` and `_∷_` constructors here? Or is it just for clarity?

Reply:
`Data.Vec.Bounded.Base` redefines `[]` and `_∷_` as *functions* so you
have constructor vs. functions and cannot disambiguate the overloaded names
on the RHS if you happen to import `Vec`'s constructors too.